mmedit/models/base_models/average_model.py

```diff
@@ -230,8 +230,8 @@ def avg_func(self, averaged_param: Tensor, source_param: Tensor,
             steps (int): The number of times the parameters have been
                 updated.
         """
-        momentum = self.rampup(self.steps, self.ema_kimg, self.ema_rampup,
-                               self.batch_size, self.eps)
+        momentum = 1. - self.rampup(self.steps, self.ema_kimg, self.ema_rampup,
```

Comment:
Since momentum is a large value before, so we need to dd a warning, refer to https://github.com/open-mmlab/mmengine/blob/bf369da22ada3336e8f44e23c7de3f3117b9c84a/mmengine/model/averaged_model.py#L176